shared/gh/js/views/gh.admin-batch-edit.js

```diff
@@ -414,7 +414,7 @@ define(['gh.core', 'gh.constants', 'gh.utils', 'moment', 'gh.calendar', 'gh.admi
         if (!triposData) {
             utils.getTriposStructure(function(err, data) {
                 if (err) {
-                    return utils.notification('Fetching triposes failed.', 'An error occurred while fetching the triposes.', 'error');
+                    return gh.utils.notification('Could not fetch triposes', constants.messaging.default.error, 'error');
```

Comment:
We might want to use `gh.utils` throughout the entire code to be consistent (and remove `utils` form the requires above).